### ISSUE TITLE
Strange Error

### ISSUE THREAD

**Issue by capriciousduck** [labbots/google-drive-upload#77]
I am getting the following error:

`/dev/fd/63: line 1: 400:: command not found`

**Comment by norbertschuler** [comment]
I once got that error as I tried to execute `gsync` or `gupload` which where not downloaded correctly from the `install.sh` script. The files instead contained some error strings.

**Comment by Akianonymus** [comment]
> I am getting the following error:
> 
> `/dev/fd/63: line 1: 400:: command not found`

You are getting this while updating ? use -D flag and provide logs

**Comment by Akianonymus** [comment]
@norbertschuler Logs would be helpful

**Comment by capriciousduck** [comment]
Here is the debug:

`root@zeus:~# gupload --debug -u
+ _update
+ declare job=update
+ [[ update =~ uninstall ]]
+ :
+ _print_center justify 'Fetching update script..' -
+ [[ 3 = 3 ]]
+ printf '%s\n' 'Fetching update script..'
Fetching update script..
+ [[ -f /root/.google-drive-upload/google-drive-upload.info ]]
+ source /root/.google-drive-upload/google-drive-upload.info
++ REPO=labbots/google-drive-upload
++ COMMAND_NAME=gupload
++ SYNC_COMMAND_NAME=gsync
++ INSTALL_PATH=/root/.google-drive-upload/bin
++ CONFIG=/root/.googledrive.conf
++ TYPE=release
++ TYPE_VALUE=latest
++ SHELL_RC=/root/.bashrc
++ LATEST_INSTALLED_SHA=bdb9d5b067549d744a3472b3b9929b980e1f0854
+ declare repo=labbots/google-drive-upload type_value=latest
+ [[ release = branch ]]
+ declare latest_sha script
++ _get_latest_sha release latest labbots/google-drive-upload
++ declare LATEST_SHA
++ case "${1:-${TYPE}}" in
++++ curl -L --compressed -s https://github.com/labbots/google-drive-upload/releases/latest
++++ grep '="/labbots/google-drive-upload/commit' -m1
+++ hash='          <a href="/labbots/google-drive-upload/commit/bdb9d5b067549d744a3472b3b9929b980e1f0854" class="muted-link">'
++ LATEST_SHA=
++ printf %b ''
+ latest_sha=
++ curl --compressed -Ls https://raw.githubusercontent.com/labbots/google-drive-upload//install.sh
+ script='400: Invalid request'
+ _clear_line 1
+ :
+ bash /dev/fd/63 --skip-internet-check
++ printf '%s\n' '400: Invalid request'
/dev/fd/63: line 1: 400:: command not found
+ exit 127`

**Comment by norbertschuler** [comment]
> @norbertschuler Logs would be helpful

Sorry, my problem got already fixed with the latest release v2.7 - I just thought mentioning this could be helpful.

**Comment by Akianonymus** [comment]
> Here is the debug:
> 
> `root@zeus:~# gupload --debug -u
> 
> * _update
> * declare job=update
> * [[ update =~ uninstall ]]
> * :
> * _print_center justify 'Fetching update script..' -
> * [[ 3 = 3 ]]
> * printf '%s\n' 'Fetching update script..'
>   Fetching update script..
> * [[ -f /root/.google-drive-upload/google-drive-upload.info ]]
> * source /root/.google-drive-upload/google-drive-upload.info
>   ++ REPO=labbots/google-drive-upload
>   ++ COMMAND_NAME=gupload
>   ++ SYNC_COMMAND_NAME=gsync
>   ++ INSTALL_PATH=/root/.google-drive-upload/bin
>   ++ CONFIG=/root/.googledrive.conf
>   ++ TYPE=release
>   ++ TYPE_VALUE=latest
>   ++ SHELL_RC=/root/.bashrc
>   ++ LATEST_INSTALLED_SHA=bdb9d5b067549d744a3472b3b9929b980e1f0854
> * declare repo=labbots/google-drive-upload type_value=latest
> * [[ release = branch ]]
> * declare latest_sha script
>   ++ _get_latest_sha release latest labbots/google-drive-upload
>   ++ declare LATEST_SHA
>   ++ case "${1:-${TYPE}}" in
>   ++++ curl -L --compressed -s https://github.com/labbots/google-drive-upload/releases/latest
>   ++++ grep '="/labbots/google-drive-upload/commit' -m1
>   +++ hash='          ['
>   ++ LATEST_SHA=
>   ++ printf %b ''](/labbots/google-drive-upload/commit/bdb9d5b067549d744a3472b3b9929b980e1f0854)
> [ ](/labbots/google-drive-upload/commit/bdb9d5b067549d744a3472b3b9929b980e1f0854)
> * [latest_sha=
>   ++ curl --compressed -Ls ](/labbots/google-drive-upload/commit/bdb9d5b067549d744a3472b3b9929b980e1f0854)https://raw.githubusercontent.com/labbots/google-drive-upload//install.sh
> * script='400: Invalid request'
> * _clear_line 1
> * :
> * bash /dev/fd/63 --skip-internet-check
>   ++ printf '%s\n' '400: Invalid request'
>   /dev/fd/63: line 1: 400:: command not found
> * exit 127`

Should be ok after https://github.com/labbots/google-drive-upload/pull/78 is merged and a new release is created.

Although, you have to run the original command to update, just once, after a new release is created, which is..

```
bash <(curl --compressed -s https://raw.githubusercontent.com/labbots/google-drive-upload/master/install.sh)
```
